Jenkinsfile

```diff
@@ -82,6 +82,24 @@ pipeline {
         ])
       }
     }
+
+    stage('Rename jenkins workspace path') {
+      agent {label 'nixos-mayastor'}
+      steps{
+        sh """
+        printenv
+        if [[ "${env.WORKSPACE}" =~ "@" ]];
+        then
+            newpath=\$(echo ${env.WORKSPACE} | sed s/"@"/"__"/)
+            mv "${env.WORKSPACE}" newpath
+            echo "Inside if"
```

Comment:
I think you should set `env.WORKSPACE`  to `newpath` for this to work properly

Reply:
I had that doubt , whether I do that or change the dir name on disk. The problem is I wouldn't know unless the PR is merged.